src/Language/PureScript/Errors.hs

```diff
@@ -348,6 +349,7 @@ errorCode em = case unwrapErrorMessage em of
   UnsupportedRoleDeclaration {} -> "UnsupportedRoleDeclaration"
   RoleDeclarationArityMismatch {} -> "RoleDeclarationArityMismatch"
   DuplicateRoleDeclaration {} -> "DuplicateRoleDeclaration"
+  CannotUnwrapHiddenNewtypeConstructor {} -> "CannotUnwrapHiddenNewtypeConstructor"
```

Comment:
Since you explicitly invited bikeshedding, how about `MissingConstructorImportForCoercible`? It's an issue for both wrapping and unwrapping, and the error names seem to prefer describing a thing to be corrected over a thing the compiler failed to do.

Reply:
Yeah that’s better, thank you for the suggestion!